### ISSUE TITLE
Is there a tool to translate arm assenmbly code to llvm-ir?

### ISSUE THREAD

**Issue by chlizheng** [trailofbits/remill#310]
Hi,
I  want to translate arm assembly code such as cortex-m3 or cortex-m0 to llvm-ir, and this project smilarly don't support it? Do you try it? And Is there good reference to do the translation?
best regards.

**Comment by mike-myers-tob** [comment]
Correct. Remill does not yet support lifting "classic ARM" instructions like those used in ARMv6-M (Coretex-M0) or ARMv7-M (Coretex-M3). 

Remill currently only supports the ARM 64-bit instruction set (AArch64) used by [ARMv8-A](https://en.wikipedia.org/wiki/ARM_architecture#ARMv8-A).

The list of ARM AArch64 instructions currently lifted by Remill is [listed here](https://github.com/trailofbits/remill/blob/master/tests/AArch64/Tests.S#L260).
